scripts/qiita-env

```diff
@@ -155,5 +157,28 @@ def job_details(job):
             click.echo("%s : %s" % (k, value))
 
 
+@env.command(name="create-portal")
+@click.option('--portal', '-p', required=True, type=str,
+              help='The portal name')
+@click.option('--desc', required=True, type=str,
+              help='The portal description')
+def add_portal(portal, desc):
+    """Creates a new portal on the database"""
+    try:
+        Portal.create(portal, desc)
+    except QiitaDBDuplicateError:
+        raise click.BadParameter("Portal name already exists!")
+
+
+@env.command(name="remove-portal")
+@click.option('--portal', '-p', required=True, type=str,
```

Comment:
Tried, but since this is qiita-env, you can possibly have a broken database when calling the commands, so having `Portal.list_portals()` can throw an error if database is not set up correctly/test database is not in a consistent state. Found this out when I called `qiita-env clean_test`